inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_shapeof.cpp

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "vpu/ngraph/transformations/dynamic_to_static_shape_shapeof.hpp"
+#include "vpu/ngraph/transformations/eliminate_shapeof_after_dsr.hpp"
```

Comment:
Nitpick: rename the current file as well `dynamic_to_static_shape_shapeof.cpp` -> `eliminate_shapeof_after_dsr.cpp`

Reply:
Fixed